Action/ShowAction.php

```diff
@@ -1,30 +0,0 @@
-<?php
-
-namespace Pim\Bundle\CustomEntityBundle\Action;
-
-use Symfony\Component\OptionsResolver\OptionsResolver;
-
-/**
- * @author    Antoine Guigan <antoine@akeneo.com>
- * @copyright 2013 Akeneo SAS (http://www.akeneo.com)
- * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
- */
-class ShowAction extends AbstractViewableAction
```

Comment:
Please don't remove action classes.

First it's a BC and secondly, it is far better than using a big controller